### DISCUSSION TITLE
When using value transform on column, column alignment retains base type norm

### DISCUSSION THREAD

**Discussion by keithdtyler** [dbeaver/dbeaver#19653] [Q&A] [answered]
### Description

Example: You have a column that contains Unix timestamps. You want them to be displayed as dates. So you use View Format -> Set Transform -> Epoch Time. Now your time_ts look like RFC 822s. Great.

The thing is, it gets hard to compare them because the formatting of the column is still (it seems) based on the underlying numeric type. So while a Y-M-D H:M:S would normally be aligned left, your column values are aligned right because that's numeric alignment.

<img width="292" alt="image" src="https://user-images.githubusercontent.com/800942/224300720-ce774627-1ec8-4fe7-9c20-c1dfd635cfe5.png">

### DBeaver Version

CE 23.0.0

### Database and driver

Scylla 5.1.6 using Cassandra driver

**Comment by uslss** [comment, marked as answer]
@keithdtyler, thanks for the report.
Unfortunately, I cannot reproduce it.
Do you have highlighted options disabled or enabled in preferences?
![image](https://user-images.githubusercontent.com/31996417/224683018-132b1388-b080-491a-9380-712e8fd553c8.png)

**Comment by alr888rav** [comment]
After I set default format for dates, I have similar problem with date alignment. Before - dates align left, after - dates jump to right. 
Change setting to off return dates to left.

Could you set default value for Right-justify date to off in future versions? Right alignment for date looks bad.

ps DBeaver CE 24.0.1, LMDE 6, database ORACLE 11.2
